pkg/endpoint/endpoint.go

```diff
@@ -964,6 +964,14 @@ func (e *Endpoint) Update(cfg *models.EndpointConfigurationSpec) error {
 
 	e.getLogger().WithField("configuration-options", cfg).Debug("updating endpoint configuration options")
 
+	// We need to make sure DatapathConfiguration.DisableSipVerification value is same as
+	// the value of SourceIPVerification option in the endpoint.
+	if e.GetOptions().IsEnabled(option.SourceIPVerification) {
+		e.DatapathConfiguration.DisableSipVerification = false
+	} else {
+		e.DatapathConfiguration.DisableSipVerification = true
+	}
```

Comment:
I don't think this change is needed. Since you removed the code in config.go, we never directly generate the macros from `e.DatapathConfiguration.DisableSipVerification`.

Instead, thanks to your change in `endpoint.go`, `e.DatapathConfiguration.DisableSipVerification` is only used as the default value for the per-endpoint runtime configuration. Then, the macros are always generated from that per-endpoint runtime configuration.

Reply:
I agree with Paul, this part is not needed, I suggest removing it and leaving as is.